src/js/utils/__tests__/mixins-test.js

```diff
@@ -0,0 +1,36 @@
+import { parseMetricToNum } from '../mixins';
+
+afterAll(() => {
+  jest.clearAllMocks();
+});
+
+describe('Mixins parseMetricToNum', () => {
+  global.console = {
+    warn: jest.fn(),
+  };
+
+  test('coverts "12px" to 12', () => {
```

Comment:
typo converts :) same with the other tests :)

Reply:
Fixed.